pkg/sstable/sstable_naming.go

```diff
@@ -42,6 +42,21 @@ func ExtractID(sstable string) string {
 	panic(unknownSSTableError(sstable))
 }
 
+// Crc32FileNameFromGivenSSTableFile returns name of the .crc32 file generated for the SSTable which contains
+// given file.
+func Crc32FileNameFromGivenSSTableFile(sstable string) string {
+	// Split the filename by dashes
+	parts := strings.Split(sstable, "-")
+
+	if len(parts) < 2 {
+		return sstable + "-Digest.crc32"
+	}
+	// Replace the last part with "Digest.crc32"
+	parts[len(parts)-1] = "Digest.crc32"
+	// Join the parts back together with dashes
+	return strings.Join(parts, "-")
+}
+
```

Comment:
Wouldn't it be safer to run it through sstable format regex?

Reply:
to check if the sstable follows name pattern ?

Reply:
I don't care if it follows the name pattern for sstable, just want to change the postfix. But maybe I should ?

Reply:
Not sure if that was the case before, but after the changes, it looks like this function is only used in tests.
Could we move it to the test pkg as well?
Then I'm fine with not using the regex - although it would still be safer.

Reply:
actually, you are right, moved to _test